packages/nextjs/app/builders/0xB24023434c3670E100068C925A87fE8F500d909a/page.tsx

```diff
@@ -0,0 +1,101 @@
+"use client";
+
+import Image from "next/image";
+import Link from "next/link";
+import { AddressCopyIcon } from "../../../components/scaffold-eth/Address/AddressCopyIcon";
+import type { NextPage } from "next";
+import { CodeBracketIcon, UserCircleIcon } from "@heroicons/react/24/outline";
+
+const MRWICKSProfile: NextPage = () => {
+  const walletAddress = "0xB24023434c3670E100068C925A87fE8F500d909a";
+
+  const socialLinks = [
+    { href: "https://x.com/mrrwiicks", label: "X", icon: CodeBracketIcon },
+    { href: "https://github.com/mrwicks00", label: "GitHub", icon: CodeBracketIcon },
+    { href: "https://discord.com/users/987567935503597578", label: "Discord", icon: UserCircleIcon },
+  ];
```

Comment:
Since these are all constant, it makes sense to place them outside of the component. So they're created only once at module initialization, rather than every render.